pages/docs/guides/message-validation.md

```diff
@@ -0,0 +1,125 @@
+---
+title: Messages validation
+description: This guide explains different use cases for message validation using AsyncAPI.
+weight: 120
+---
+
```

Comment:
Heyo @starlightknown, I'm noticing a discrepancy in whether or not `schema registry` is capitalized. 

Please go through this entire doc again and fix all mentions of `Schema Registry` by capitalizing them. ✌🏽